docs/concepts/validator-faq.md

```diff
@@ -249,4 +249,4 @@ That means you have **Lost Your Validator Forever!** You can only create a new o
 ## Community Channels
 
 - Riot chat: <https://riot.im/app/#/room/#irisvalidators:matrix.org>
-- IRIShub Validator Working Group QQ Group：834063323
+- IRIShub Validator Working Group QQ Group: 834063323
```

Comment:
add `discord` channel ?

Reply:
I think it is better to maintain all contact information in another PR